src/parquet/arrow/writer.cc

```diff
@@ -51,6 +51,22 @@ using parquet::schema::GroupNode;
 namespace parquet {
 namespace arrow {
 
+constexpr int64_t JULIAN_EPOCH_OFFSET_DAYS = 2440588LL;
```

Comment:
Google C++ style guide requires constants to be named like kJulianEpochOffsetDays